src/components/profile/latestQuestions/LatestQuestions.jsx

```diff
@@ -0,0 +1,13 @@
+import Question from '../../Question/Question';
+
+function LatestQuestions() {
+  return (
+    <div>
+      <Question />
```

Comment:
Is this component not getting any data as props. Like questions or anything?

Reply:
This component gets its data from the API when we fetch the incoming data.